js/script.js

```diff
@@ -7,7 +7,6 @@ buttons.forEach((button) => {
       this.parentElement.parentElement.classList.remove("active");
       return;
     }
-    accordions.forEach((accord) => accord.classList.remove("active"));
     this.parentElement.parentElement.classList.add("active");
```

Comment:
If one element is open, the other must be closed. Please add a transition when it opens and closes.